src/main/resources/templates/tools.html

```diff
@@ -31,13 +32,12 @@ <h4 align="left"><strong>Ready-to-use Common Vulnerabilities and Exposure CVE En
                 <li>A description of known vulnerabilities that can be found on  the internet.</li>
                 <li>Ready-to-use CVEs are deployable environments where the vulnerability exists and are difficult to build. </li>
             </ul>
-            <p align="left"><strong>Potential Users</strong></p>
-            <ul>
-                <li>System Security researchers &amp;lecturers</li>
-            </ul>
-             <hr/>
-            </div>
+            <!--<p align="left"><strong>Potential Users</strong></p>-->
+            <!--<ul>-->
+                <!--<li>System Security researchers &amp;lecturers</li>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)